ros/Makefile

```diff
@@ -14,31 +22,75 @@ deps:
 		--ignore-src
 
 .PHONY: build
-build:
-	colcon build
+build: cpp-sdk
+	colcon build --cmake-args -DUSE_LOCAL_PREBUILT_SDK=$(USE_LOCAL_PREBUILT_SDK)
+
+.PHONY: build-msgs
+build-msgs:
+	colcon build --packages-select foxglove_msgs
+
+.PHONY: cpp-sdk
+cpp-sdk:
+ifeq ($(BUILD_SDK), ON)
+	$(info Building Foxlove C++ SDK from local source tree)
+	make -C ../cpp
+endif
+
+ifeq ($(USE_LOCAL_PREBUILT_SDK), ON)
+	$(info Using pre-built Foxglove C++ SDK binaries from local source tree)
+else
+	$(info Build will use pre-built Foxglove C++ SDK binaries)
+endif
+
+.PHONY: build-bridge
+build-bridge: cpp-sdk
```

Comment:
is there value in separating out the build bridge vs build msgs as separate make targets? people can always run colcon manually if they want, so it might be easier if our helper make targets just build all ros packages in the repo together?

Reply:
👍 I was erring on the side of more granular build targets, but agree that leaving it to power users to target their colcon builds is better from a maintainability standpoint